Toolset/palettes/inspector/editors/com.livecode.pi.gradientramp.behavior.livecodescript

```diff
@@ -87,6 +87,7 @@ on updateColorValue pColor
    lock screen
    lock messages
    set the swatchColor of widget "color" of me to pColor
+   put pColor into field "value" of me
```

Comment:
Unfortunately I'm not quite sure why there are two such similar implementations! 

Originally I was using separate implementations for updating the visual feedback of the chosen color value and updating the stored editor value - it looks like in the course of fixing things these two handlers have become effectively identical, so I think there is no problem with merging them into one handler.